sdk/storage/azblob/service/client_test.go

```diff
@@ -711,6 +720,78 @@ func (s *ServiceUnrecordedTestsSuite) TestSASServiceClient() {
 	testcommon.ValidateBlobErrorCode(_require, err, bloberror.AuthenticationFailed)
 }
 
+func (s *ServiceUnrecordedTestsSuite) TestSASServiceClientNoKey() {
```

Comment:
Can these tests be converted to recorded mode?

Reply:
done